packages/config/config/devices/0x027a/zen05.json

```
@@ -0,0 +1,83 @@
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN05",
+	"description": "Outdoor Smart Plug",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xb001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "1.1",
```

Unless this is used to split a device into multiple files, min should be 
```suggestion
		"min": "0.0",
```